UPGRADING.md

```diff
@@ -1,14 +1,27 @@
 Upgrading
 =========
 
+## 3.x to 4.x
+
+#### Setting Endpoints
+
+The way endpoints will be configured has been change slightly, in order to make sure all of your requests get send to the correct place.  You'll no longer be able to pass the `endpoint` parameter to your `Bugsnag\Client::make` calls, and should instead use the [`setEndpoints` configuration option](https://docs.bugsnag.com/platforms/php/other/configuration-options/#endpoints).  If you've previously used the `BUGSNAG_ENDPOINT` environment variable, it's now been renamed to `BUGSNAG_NOTIFY_ENDPOINT`, and will only work in conjunction with the `BUGSNAG_SESSION_ENDPOINT` environment variable, to make sure that both endpoints are being used correctly.
```

Comment:
A code sample to show a diff between how it used to work and how it works now would be helpful.

```diff
- old way
+ new way
```